docs/markdown/20-interactions/01-contexts.md

```diff
@@ -15,6 +15,8 @@ Your action will be run inside a workflow and having access to its contexts.
 <!-- .element: class="list-fragment" -->
 
 [Contexts Reference](https://docs.github.com/en/actions/learn-github-actions/contexts)
+/
```

Comment:
this "/" look like a missclick

Reply:
Not a missclick 😄 
<img width="481" alt="Capture d’écran 2022-10-22 à 20 07 38" src="https://user-images.githubusercontent.com/1280212/197356126-6a66914f-937b-4dd2-b55c-ff8fe704efa7.png">

Reply:
ohh okey should be good with one "/" so